pettingzoo/classic/rlcard_envs/leduc_holdem.py

```diff
@@ -82,42 +74,61 @@
 """
 from __future__ import annotations
 
+import os
+
 import gymnasium
-from rlcard.utils.utils import print_card
+import numpy as np
+import pygame
+from gymnasium.utils import EzPickle
 
 from pettingzoo.classic.rlcard_envs.rlcard_base import RLCardBase
 from pettingzoo.utils import wrappers
 
+def get_image(path):
+    from os import path as os_path
+
+    cwd = os_path.dirname(__file__)
+    image = pygame.image.load(cwd + "/" + path)
+    return image
+
+
+def get_font(path, size):
+    from os import path as os_path
+
+    cwd = os_path.dirname(__file__)
+    font = pygame.font.Font((cwd + "/" + path), size)
+    return font
+
 
 def env(**kwargs):
-    render_mode = kwargs.get("render_mode")
-    if render_mode == "ansi":
-        kwargs["render_mode"] = "human"
-        env = raw_env(**kwargs)
-        env = wrappers.CaptureStdoutWrapper(env)
-    else:
-        env = raw_env(**kwargs)
+    env = raw_env(**kwargs)
     env = wrappers.TerminateIllegalWrapper(env, illegal_reward=-1)
     env = wrappers.AssertOutOfBoundsWrapper(env)
     env = wrappers.OrderEnforcingWrapper(env)
     return env
 
 
-class raw_env(RLCardBase):
+class raw_env(RLCardBase, EzPickle):
     metadata = {
-        "render_modes": ["human"],
+        "render_modes": ["human", "rgb_array"],
         "name": "leduc_holdem_v4",
         "is_parallelizable": False,
         "render_fps": 1,
     }
 
     def __init__(
         self,
-        num_players: int = 2,
+        # num_players: int = 2, # Ludec only allows fixed number of players = 2
```

Comment:
Just remove this argument if there's no number of players options, cleaner than having it commented out

Reply:
Done